pipeline/src/node_properties/model_inference_properties.rs

```diff
@@ -45,8 +47,16 @@ pub struct ModelInferenceRuntime {
     pub infer_node_unique_ids: Option<BTreeMap<String, i32>>,
 }
 
+#[skip_serializing_none]
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct ClassInferenceProperties {
     /// Minimum inference threshold, should be set on the [0.0, 1.0] interval.
-    pub min_inference_threshold: f32,
+    pub min_inference_threshold: Option<f32>,
+    /// Relative difference between sides of the rectangles to merge them into a group
+    /// Used in OpenCV groupRectangles function and DBSCAN algorithm
```

Comment:
Tiny nit: When the docs consist of multiple sentences the first one should be a "summary" line and be on its own paragraph.

```suggestion
    /// Relative difference between sides of the rectangles to merge them into a group
    ///
    /// Used in OpenCV groupRectangles function and DBSCAN algorithm
```

(and maybe add periods at the end of the sentences everywhere)